lib/v2/screens/profile_screens/security/security_screen.dart

```diff
@@ -51,6 +51,11 @@ class SecurityScreen extends StatelessWidget {
                   ),
                 );
               },
+            ),
+            BlocListener<SecurityBloc, SecurityState>(
+              listenWhen: (previous, current) =>
+                  previous.navigateToGuardians == false && current.navigateToGuardians == true,
```

Comment:
this is fine for this page. 

For larger pages we will use `val pageCommand PageCommand?`

If the page command is null, dont listen, if its not null listen and then execute the command. 

Once the command is execute we send an intent to reset the command back to null

Reply:
You mean creating an event just to reset the variable to false again, something like this?

```
            BlocListener<SecurityBloc, SecurityState>(
              listenWhen: (previous, current) =>
                  previous.navigateToGuardians == false && current.navigateToGuardians == true,
              listener: (context, state) {
                BlocProvider.of<SecurityBloc>(context).add(const ResetNavigateToGuardians());
                NavigationService.of(context).navigateTo(Routes.guardianTabs);
              },
            )
```

Reply:
Yeah, exactly this. 

For this case it doesnt really make much sense. 

But there are cases in more complex pages where you end up with many commands, then it makes sense.